mscore/data/shortcuts.xml

```diff
@@ -527,7 +527,7 @@
     <seq>F11</seq>
     </SC>
   <SC>
-    <key>toggle-navigator</key>
+    <key>toggle-timeline</key>
```

Comment:
Hmm, OK... Guess I'd need to see it in action, but I do like (and often use) the navigator